android/app/build.gradle

```diff
@@ -227,7 +239,7 @@ dependencies {
     debugImplementation("com.facebook.flipper:flipper-fresco-plugin:${FLIPPER_VERSION}") {
         exclude group: 'com.facebook.flipper'
     }
-    androidTestImplementation(project(path: ":detox"))
+    // androidTestImplementation(project(path: ":detox"))
```

Comment:
Why is this commented out?

Reply:
I just forgot to delete it. It's commented out because I'm using `androidTestImplementation('com.wix:detox:+')` in the line below